### PR TITLE
Readme changes for SageMaker Studio

### PR DESCRIPTION
**Pull request by akartsky**
Readme changes for SageMaker Studio

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/content/docs/tutorials/sagemaker-example.md [hunk 1]
```diff
@@ -102,14 +102,47 @@ echo $IAM_ROLE_ARN_FOR_IRSA
 
 For more information on authorization and access for ACK service controllers, including details regarding recommended IAM policies, see [Configure Permissions][configure-permissions].
 
+#### Optional step
```
Comment (surajkota) [inline docs/content/docs/tutorials/sagemaker-example.md:105]: let's make it part of the regular steps. People might miss optional step and then wonder why studio is not working

File: docs/content/docs/tutorials/sagemaker-example.md [hunk 2]
```diff
@@ -102,14 +102,47 @@ echo $IAM_ROLE_ARN_FOR_IRSA
...
+
+If you are planning to try out the Domain, UserProfile or App resource then `AmazonSageMakerFullAccess` won't be enough.
```
Comment (surajkota) [inline docs/content/docs/tutorials/sagemaker-example.md:107]: nit: reword

`AmazonSageMakerFullAccess` policy does not include permissions required to manage Domain, UserProfile or App resource. Create and attach the following Policy to the IAM Role to ensure that your SageMaker service controller has access to these resources.

File: docs/content/docs/tutorials/sagemaker-example.md [hunk 3]
```diff
@@ -102,14 +102,47 @@ echo $IAM_ROLE_ARN_FOR_IRSA
...
+printf '{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "SagemakerAllAccess",
+      "Effect": "Allow",
+      "Action": [
+        "sagemaker:*"
+      ],
+      "Resource": "*"
```
Comment (surajkota) [inline docs/content/docs/tutorials/sagemaker-example.md:120]: more control if possible

File: docs/content/docs/tutorials/sagemaker-example.md [hunk 4]
```diff
@@ -102,14 +102,47 @@ echo $IAM_ROLE_ARN_FOR_IRSA
...
+    }
+  ]
+}
+' > ./sagemaker_all_access.json
```
Comment (surajkota) [inline docs/content/docs/tutorials/sagemaker-example.md:124]: nit: sagmaker_studio_access.json

File: docs/content/docs/tutorials/sagemaker-example.md [hunk 5]
```diff
@@ -102,14 +102,47 @@ echo $IAM_ROLE_ARN_FOR_IRSA
...
+```
+
+```bash
+aws --region $SERVICE_REGION iam put-role-policy --role-name $OIDC_ROLE_NAME --policy-name SagemakerAllAccess --policy-document file://./sagemaker_all_access.json
```
Comment (surajkota) [inline docs/content/docs/tutorials/sagemaker-example.md:128]: `aws --region $SERVICE_REGION iam put-role-policy --role-name $OIDC_ROLE_NAME --policy-name SagemakerStudioAccess --policy-document file://sagmaker_studio_access.json`

**Review by surajkota — CHANGES_REQUESTED** [review]

File: docs/content/docs/tutorials/sagemaker-example.md (hunk 1 above)
Comment (akartsky) [reply docs/content/docs/tutorials/sagemaker-example.md]: removed

File: docs/content/docs/tutorials/sagemaker-example.md (hunk 2 above)
Comment (akartsky) [reply docs/content/docs/tutorials/sagemaker-example.md]: thanks !

File: docs/content/docs/tutorials/sagemaker-example.md (hunk 3 above)
Comment (akartsky) [reply docs/content/docs/tutorials/sagemaker-example.md]: done

**Review by surajkota — APPROVED** [review]
/approve
/lgtm

**Comment by surajkota** [comment]
@aws-controllers-k8s/community-maintainers ready to merge. Please take a look

**Pull request merged**